src/common/services/UtilService/UtilService.ts

```diff
@@ -1,5 +1,7 @@
+import moment from 'moment';
 import generateUUID from 'react-native-uuid';
-
 export class UtilService {
   uuid = (): string => generateUUID.v1();
+
+  now = (): number => moment().unix();
```

Comment:
`dateNow` or `timeNow` since it's `utils.now()` it's a bit ambiguous (different story if it was `dateUtils`- which i'm not against but probably overkill)